azavea/raster-foundry
# Rename buckets to projects

## Overview

Rename Buckets to Projects

### Checklist


## Testing Instructions
* Run migrations
* Make a project:
  * Select scenes in the browse view, click "Add to Project", then create a new one in the modal
  * In the `/library/projects` view, create a new project
* Delete a project
   * Go to a project scene list at `/library/projects/detail/{uuid}/scenes and click "Delete Project"
* Edit a project
  * _Broken_: created #742 to fix, since it wasn't caused by changes (never implemented)
* Add scenes to a project
  * Browse view:
    * Select scenes by clicking the "plus" icon buttons
    * Click "Add to Project" at the bottom of the side panel
    * Select projects to add the scenes to in the modal
    * Verify that scenes were added by viewing the project details at `library/projects/{uuid}/scenes` (library can be found in the user dropdown menu on the right side of the navbar)
* Remove scenes from a project
  * In `library/projects/{uuid}/scenes`, select one / several scenes and remove them from the project
* Keep a watch out for javascript errors in chrome dev console while doing the above

Discovered #742, #743
